docs/codacy-api/api-tokens.md

```diff
@@ -46,7 +46,7 @@ You can create new project API tokens programmatically using the Codacy API endp
 1.  Click the button **Settings** on the **Project API** integration and copy the project API token.
 
     !!! tip
-        You can create multiple project API tokens. This can be useful to have a more flexible control by revoking only a specific token.
+        You can create up to 100 project API tokens per repository. This can be useful to have a more flexible control by revoking only a specific token.
```

Comment:
```suggestion
        You can create multiple (up to 100) project API tokens per repository. This can be useful to have a more flexible control by revoking only a specific token.
```

Reply:
or maybe having it in a separate `!!! tip` section